getting-started-with-recoil-in-react-1fca/notes/01_react-create-shared-state-with-recoil-in-a-react-app.md.md

```diff
@@ -0,0 +1,38 @@
+<TimeStamp start="1:17" end="1:27">
+  
+  ```html
+    <button style={{
+      fontSize: 100,
+      cursor: 'pointer',
+      background: 'transparent',
+      border: 0
+    }}>
+      🧻
+    </button>
+  ```
+  
+</TimeStamp>
+
+<TimeStamp start="1:49" end="1:55">
+  
+  [recoiljs.org - Atoms](https://recoiljs.org/docs/basic-tutorial/atoms/)
+  
+</TimeStamp>
+
+<TimeStamp start="2:02" end="2:12">
+  
+  Atoms allow you to share state across components
+  
+</TimeStamp>
+
+<TimeStamp start="2:20" end="2:27">
+  
+  Atom keys must be unique
+  
+</TimeStamp>
+
+<TimeStamp start="2:46" end="2:54">
+  
+  useRecoilState is required to use atoms. [recoiljs.org - useRecoilState](https://recoiljs.org/docs/api-reference/core/useRecoilState/)
```

Comment:
I think having a definition here along with the link to the docs is a good way to go. Save the users from having to click on the docs if they don't want to.